README.md

```diff
@@ -188,12 +188,12 @@ password empty.
 **Note**: _We STRONGLY suggest, not to use this, even if this add-on is
 only exposed to your internal network. USE AT YOUR OWN RISK!_
 
-## Using the Pi-hole sensor in Home Assistant
+## Using the Pi-hole integration in Home Assistant
 
-Home Assistant offers a [Pi-hole sensor][pi-hole-sensor] that allows you to
-display the statistical summary of your Pi-hole installation.
+Home Assistant offers a [Pi-hole integration][pi-hole-integration] that allows you to
```

Comment:
This will not work, you've changed the anchors, but haven't adjust them.
This breaks the links. Please adjust.

Reply:
I did this to prevent merge error with https://github.com/hassio-addons/addon-pi-hole/pull/102

Reply:
That one is not merged and will be reviewed separately. Instead of stacking PR's you can also leave review comments. Closing PR.